NAMESPACE

```diff
@@ -283,6 +285,7 @@ export(row_means)
 export(row_to_colnames)
 export(rowid_as_column)
 export(rownames_as_column)
+export(seek_variables)
```

Comment:
I think we should avoid too many aliases, it doesn't seem necessary to have one here (no pref on the one to keep though)

Reply:
Agree. The idea for `data_seek()` came later, and that one is more in line with our other function names. I still would prefer `data_seek()` over `data_search()`, though, because _seek_ sounds more like investigation than _search_ to me, but... I'm not a native speaker ;-)

Reply:
Well me neither :smile: but of course there's a stackexchange for everything: https://english.stackexchange.com/questions/35507/seek-vs-search

> One fundamental difference is that the object of "seek" is the item you are trying to locate, whereas the object of "search" is the place you are looking

So based on this I would go with `seek` as well since we look for a specific regex